Mathlib/CategoryTheory/Abelian/GrothendieckCategory/Colim.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Abelian.Basic
+import Mathlib.CategoryTheory.Abelian.GrothendieckAxioms.Basic
+
+/-!
+# Exactness of colimits in abelian categories
+
+Assuming `HasExactColimitsOfShape J C`, we rephrase the exactness of
+`colim : (J ⥤ C) ⥤ C` by saying that if `S : ShortComplex (J ⥤ C)`
+is exact, and we have colimit cocones `c₁`, `c₂`, `c₃`
+for `S.X₁`, `S.X₂` and `S.X₃`, then the induced short complex
+`c₁.pt ⟶ c₂.pt ⟶ c₃.pt` is exact.
+
+-/
+
+universe v v' u u'
+
+namespace CategoryTheory.Limits
+
+variable {C : Type u} [Category.{v} C] [Abelian C]
```

Comment:
I think the entire file isn't really about abelian categories at all, the `Abelian C` can be replaced by `HasZeroMorphisms C`. So I think it makes sense to move the file to `GrothendieckAxioms` instead of `GrothendieckCategory` and generalize the typeclass assumption.

Reply:
Thanks for the suggestion, I have moved the code into `GrothendieckAxioms.Colim` with a `HasZeroMorphisms` assumption.